src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/MoreOptionsPopOver.java

```diff
@@ -56,9 +54,10 @@ private void clickReportLinkOption(String linkText) {
     }
   }
 
-  private void waitFor(final Function<PostEntity, Boolean> function) {
-    new FluentWait<>(postEntity).withTimeout(DiscussionsConstants.TIMEOUT, TimeUnit.SECONDS)
-        .until(function);
+  private void waitFor(final Function<PostEntity, Boolean> predicate) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "waitFor" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)